src/core/adapters/autocomplete.ts

```diff
@@ -23,9 +23,13 @@ namespace Adapter {
     const searchTerms = result.searchTerms || [];
     const navigations = result.navigations || [];
     const hasCategory = category && searchTerms[0] && Adapter.termsMatch(searchTerms[0].value, query);
+    // tslint:disable-next-line max-line-length
+    const categoryValues = hasCategory ? [{ matchAll: true }, ...Adapter.extractCategoryValues(searchTerms[0], category)] : [];
+    if (hasCategory) {
```

Comment:
I don't think we test that `searchTerms` gets shifted

Reply:
New test added to check for this.